config/initializers/geocoder.rb

```diff
@@ -0,0 +1,14 @@
+# rubocop:disable Naming/VariableNumber
+Geocoder.configure(
+  ip_lookup: :geoip2,
+  geoip2: {
+    file: ENV.fetch('GEOIP_FILE', File.join('/usr', 'share', 'GeoIP', 'GeoLite2-Country.mmdb'))
```

Comment:
I've used the default location that maxmind's `geoipupdate` tool uses.

Reply:
I'm using the country database as that fits or need and is also a smaller database than the other databases (which have more information).

Reply:
Maybe add these as comments to this file as they're useful for future reference.